gc/base/TLHAllocationSupport.cpp

```diff
@@ -354,6 +364,7 @@ MM_TLHAllocationSupport::allocateTLH(MM_EnvironmentBase *env, MM_AllocateDescrip
 void
 MM_TLHAllocationSupport::flushCache(MM_EnvironmentBase *env)
 {
+	restoreTLHTopForGC(env);
```

Comment:
There is reconnect() path that does something like this. ATM, I have no clue when it's used, but possibly needs restoreTLHTopForGC, too (or better, just call flushCache.

Reply:
I remember looking into the `reconnect()` path and it wasn't a concern.

Reply:
We don't ever take the `shouldFlush` path here
```C++
 void
 MM_TLHAllocationSupport::reconnect(MM_EnvironmentBase *env, bool shouldFlush)
{
	 MM_GCExtensionsBase *extensions = env->getExtensions();

	if(shouldFlush) {
		_abandonedList = NULL;
		_abandonedListSize = 0;
		clear(env);
	} else {
		/* Clear current information accumulated */	
```

Reply:
i'm fine with removing the path